File: javascript/bee1019.js
let segundos = parseInt(require("fs").readFileSync("stdin", "utf8"));

const horas = segundos / 3600;
segundos %= 3600;
const minutos = segundos / 60;
segundos %= 60;

console.log(`${Math.floor(horas)}:${Math.floor(minutos)}:${Math.floor(segundos)}`);

File: java/bee1019.java
import java.util.Scanner;

class Tempo {
    private int segundos;

    Tempo(int segundos) {
        this.segundos = segundos;
    }

    void showTempo() {
        int horas = segundos / 3600;
        int resto = segundos % 3600;

        int minutos = resto / 60;
        resto %= 60;

        System.out.println(horas + ":" + minutos + ":" + resto);
    }
}

public class bee1019 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int segundos = input.nextInt();

        Tempo tempo = new Tempo(segundos);
        tempo.showTempo();

        input.close();
    }
 
}
